src/material/expansion/expansion-panel-header.scss

```diff
@@ -45,12 +45,23 @@
   flex: 1;
   flex-direction: row;
   overflow: hidden;
+
+  // width of .mat-expansion-indicator::after element
+  &:only-child {
```

Comment:
This might break down if we add another element to the markup. Maybe we can set a class based on whether the toggle exists instead?

Reply:
@crisbeto I've adopted this approach. I also realized that my original commit did not take into consideration `togglePosition="before"` being used. My new commit takes this into consideration.

![image](https://user-images.githubusercontent.com/18009315/87811601-eab3f580-c82c-11ea-9c52-25e6be2b4806.png)